server/src/internalClusterTest/java/org/opensearch/indices/replication/SegmentReplicationBaseIT.java

```diff
@@ -204,8 +203,7 @@ protected IndexShard getIndexShard(String node, String indexName) {
     }
 
     protected boolean segmentReplicationWithRemoteEnabled() {
```

Comment:
better to remove this function instead .

Reply:
This method is still being used in certain ITs

Reply:
we should remove it .